### ISSUE TITLE
install_freebsd_stable_deps not found

### ISSUE THREAD

**Issue by marek-knappe** [saltstack/salt-bootstrap#1414]
### Description of Issue/Question

      /tmp/kitchen/bootstrap.sh: install_freebsd_stable_deps: not found
416         * ERROR: Failed to run install_freebsd_git_deps()!!!

### Setup
Just new install on Freebsd 11

### Steps to Reproduce Issue
Just install from bootstrap on freebsd11

### LOGS
    downloading https://bootstrap.saltstack.com
390          to file /tmp/kitchen/bootstrap.sh
391        trying curl...
392          % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
393                                  Dload  Upload   Total   Spent    Left  Speed
394 100  263k  100  263k    0     0  2528k      0 --:--:-- --:--:-- --:--:-- 2528k
395         *  INFO: Running version: 2020.01.21
396         *  INFO: Executed by: shell pipe
397         *  INFO: Command line: '/tmp/kitchen/bootstrap.sh -X git v2019.2.0'
398        
399         *  INFO: System Information:
400         *  INFO:   CPU:          AMD EPYC 7571
401         *  INFO:   CPU Arch:     amd64
402         *  INFO:   OS Name:      FreeBSD
403         *  INFO:   OS Version:   11.3-RELEASE-p5
404         *  INFO:   Distribution: FreeBSD 11.3
405        
406         *  INFO: Installing minion
407         *  INFO: Daemons will not be started
408         *  INFO: Found function install_freebsd_git_deps
409         *  INFO: Found function config_salt
410         *  INFO: Found function preseed_master
411         *  INFO: Found function install_freebsd_git
412         *  INFO: Found function install_freebsd_git_post
413         *  INFO: Found function install_freebsd_restart_daemons
414         *  INFO: Found function daemons_running
415         *  INFO: Running install_freebsd_git_deps()
416        /tmp/kitchen/bootstrap.sh: install_freebsd_stable_deps: not found
417         * ERROR: Failed to run install_freebsd_git_deps()!!!
418        No salt-minion installed, install must have failed!!
419        salt_install = bootstrap
420        salt_url = https://bootstrap.saltstack.com
421        bootstrap_options = -X git v2019.2.0
422        salt_version = 2019.2.0

### Versions and Systems

395         *  INFO: Running version: 2020.01.21

**Comment by marek-knappe** [comment]
Looks like in 
```

commit ccebf6ce82f77f85a62d4ed082df8f3af0c704fe
Author: kgbsd <45101782+kgbsd@users.noreply.github.com>
Date:   Mon Nov 18 07:25:28 2019 -0700

    Update bootstrap-salt.sh

    missed a freebsd_9_stable_deps part

    Co-Authored-By: Denys Havrysh <Denys.Gavrysh@gmail.com>

diff --git a/bootstrap-salt.sh b/bootstrap-salt.sh
index f39b884..edad5e1 100755
--- a/bootstrap-salt.sh
+++ b/bootstrap-salt.sh
@@ -5280,7 +5280,7 @@ install_freebsd_deps() {
 }

 install_freebsd_git_deps() {
-    install_freebsd_9_stable_deps || return 1
+    install_freebsd_stable_deps || return 1

     # shellcheck disable=SC2086
     SALT_DEPENDENCIES=$(/usr/local/sbin/pkg search -R -d sysutils/py-salt | grep -i origin | sed -e 's/^[[:space:]]*//' | tail -n +2 | awk -F\" '{print $2}' | tr '\n' ' ')

     
```
Someone changed it from freebsd_9_stable_deps that was already not existing from commit: da991e3e1abda1777065d9deb65e1e5c3e3fc609, to non existing freebsd_stable_deps

and function name is :     install_freebsd_deps

**Comment by amontalban** [comment]
@vutny please disregard this, another of our team members @cedwards is working on a fix in #1413

**Comment by s0undt3ch** [comment]
Thank You. Closing.
